system/Commands/Generators/CreateScaffold.php

```diff
@@ -129,7 +129,7 @@ public function run(array $params)
 		];
 
 		// Call those commands!
-		$class = $params[0] ?? CLI::getOption(0);
+		$class = $params[0] ?? CLI::getOption('0');
```

Comment:
```suggestion
		$class = $params[0];
```
Good catch for this. But I think I made a bug here. this should be CLI::getArgument(0). Anyway, the params check is enough in this case.

Reply:
fixed.